stable/sonatype-nexus/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: sonatype-nexus
-version: 1.10.0
+version: 1.10.1
```

Comment:
Please bump the minor version as this adds a new feature

Reply:
Updated.